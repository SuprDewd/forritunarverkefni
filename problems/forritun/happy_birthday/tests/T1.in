Bjarni
4
kk
