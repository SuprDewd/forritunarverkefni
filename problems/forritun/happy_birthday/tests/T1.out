Hann á afmæli í dag,
hann á afmæli í dag.
Hann á afmæli hann Bjarni,
hann á afmæli í dag.

Hann er 4 ára í dag,
hann er 4 ára í dag.
Hann er 4 ára hann Bjarni,
hann er 4 ára í dag.
